src/pages/Lend/Filter/FilterSections/LocationMulti/FilterSectionLocationMulti.vue

```diff
@@ -54,14 +55,20 @@ export default {
 	},
 	methods: {
 		transformItems(items) {
+			// filter out any undefined or malformed facets ref. VUE-271
+			const filteredItems = _filter(items, item => {
+				if (typeof item !== 'undefined' && item.value !== '>') {
```

Comment:
oops, I commented on this in the merged pr already https://github.com/kiva/ui/pull/1602#discussion_r407859152
It's nothing big.